changelog.txt

```diff
@@ -0,0 +1,3 @@
+Version 1.0.11
+------------
+* Apply hotfix 2.7.9 for Mac OS to query WPJ cert using issuers from authentication challenge
```

Comment:
are we going to maintain the change log here? or we just want to rely on the change logs in the Gitnub Release page?

Reply:
yes. The changeling here helps us find which version of common core particular ADAL or MSAL shipped with. Currently you need to check the hash and lookup the hash, which is time consuming. Updated the file with previous/historic changelog.